npm.js

```diff
@@ -62,12 +62,16 @@ exports.translate = function(load){
 				packages[pkg.name+"@"+pkg.version] = true;
 			}
 		});
-		var configDependencies = ['@loader','npm-extension'].concat(configDeps.call(loader, pkg));
+		var configDependencies = [
+			'@loader',
+			'npm-extension',
+			'process'
```

Comment:
Another alternative is to detect source that needs a global process in a translate hook and update the actual source code to do something like `window.process = require("process")`